docs/01-components-and-layout.md

```diff
@@ -65,12 +65,16 @@ Override the `OnInitializedAsync` method in the `@code` block to retrieve the li
 
     protected override async Task OnInitializedAsync()
     {
-        specials = await HttpClient.GetFromJsonAsync<List<PizzaSpecial>>("specials");
+        specials = await HttpClient.GetFromJsonAsync<List<PizzaSpecial>>("specials", BlazingPizza.OrderContext.Default.ListPizzaSpecial);
     }
 }
 ```
 
-The `/specials` API is defined by the `SpecialsController` in the **BlazingPizza.Server** project.
+The `/specials` API is defined by a minimal API at `PizzaApiExtensions.cs` in the **BlazingPizza.Server** project.
+
+See also [Create a minimal web API with ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/tutorials/min-web-api?view=aspnetcore-6.0) for additional details.
+
+> Note: `BlazingPizza.OrderContext.Default.ListPizzaSpecial` refers to Json serialization with [source generators](https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-source-generation).
```

Comment:
Considering `SpecialController` no longer exist and has been replaced with a minimal API instead. I think it could be helpful to mention where that endpoint can be found, as well as adding a note for those who are not familiar with Minimal APIs

Reply:
Agreed - and I converted this to use Minimal APIs with the goal of being able to show and talk about Minimal APIs

Reply:
I think we should add a block in the documentation to help folks get started with this concept.

Reply:
That sounds great @csharpfritz